lib/internal/per_context/primordials.js

```diff
@@ -30,44 +30,86 @@ function copyProps(src, dest) {
   }
 }
 
+function getNewKey(key) {
+  return typeof key === 'symbol' ?
+    `Symbol${key.description[7].toUpperCase()}${key.description.slice(8)}` :
+    `${key[0].toUpperCase()}${key.slice(1)}`;
+}
+
 function copyPropsRenamed(src, dest, prefix) {
   for (const key of Reflect.ownKeys(src)) {
-    if (typeof key === 'string') {
-      Reflect.defineProperty(
-        dest,
-        `${prefix}${key[0].toUpperCase()}${key.slice(1)}`,
-        Reflect.getOwnPropertyDescriptor(src, key));
+    const newKey = getNewKey(key);
+    const desc = Reflect.getOwnPropertyDescriptor(src, key);
+    if ('get' in desc || 'set' in desc) {
+      const { enumerable, get, set } = desc;
+      if (get !== undefined) {
+        Reflect.defineProperty(dest, `${prefix}Get${newKey}`, {
+          value: uncurryThis(get),
+          enumerable
+        });
+      }
+      if (set !== undefined) {
+        Reflect.defineProperty(dest, `${prefix}Set${newKey}`, {
+          value: uncurryThis(set),
+          enumerable
+        });
+      }
```

Comment:
When would a setter be relevant for primordials? I'm -1 on adding this if we don't use it.